Based on:
From 000590062c55988251c2edc2467d67ad546efe50 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Mon, 24 Sep 2012 20:06:58 -0400
Subject: [PATCH] fix building with glibc-2.16

Need to include this for PRIO_xxx defines.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
diff --git a/src/afsd/afsd_kernel.c b/src/afsd/afsd_kernel.c
index 093ba5cc0..3da75050a 100644
--- a/src/afsd/afsd_kernel.c
+++ b/src/afsd/afsd_kernel.c
@@ -31,7 +31,7 @@
 #include <sys/file.h>
 #include <sys/wait.h>
 
-#if defined(AFS_LINUX20_ENV)
+#ifdef HAVE_SYS_RESOURCE_H
 #include <sys/resource.h>
 #endif
 
